lib/active_admin/async_exporter/reports/dsl.rb

```diff
@@ -4,6 +4,21 @@ module ActiveAdmin
   module AsyncExporter
     module Reports
       module DSL
+        attr_reader :csv_fields
+
+        def csv_async(decorate_model: false)
+          @csv_fields ||= {}
+
+          yield
+
+          csv_report(columns: csv_fields, decorate_model: decorate_model)
+        end
+
+        def csv_column(column_name, column_value = nil)
+          column_value ||= column_name
+          csv_fields[:"#{column_name}"] = column_value
```

Comment:
What about `column_name.to_sym` ?

Reply:
I am going to add in the README the usage of the gem with an example using symbols, I guess there's no harm with adding that little catch though, changing!